stub/stub.meta

```diff
@@ -0,0 +1,45 @@
+[ccpp-table-properties]
+  name = stub
+  type = scheme
+  dependencies =
+
+########################################################################
+[ccpp-arg-table]
+  name = stub_init
+  type = scheme
+[errmsg]
+  standard_name = ccpp_error_message
+  long_name = error message for error handling in CCPP
+  units = none
+  dimensions = ()
+  type = character
+  kind = len=*
+  intent = out
+[errflg]
+  standard_name = ccpp_error_flag
+  long_name = error flag for error handling in CCPP
+  units = flag
+  dimensions = ()
+  type = integer
+  intent = out
+
+########################################################################
+[ccpp-arg-table]
+  name = stub_finalize
+  type = scheme
+[errmsg]
+  standard_name = ccpp_error_message
+  long_name = error message for error handling in CCPP
+  units = none
+  dimensions = ()
+  type = character
+  kind = len=*
+  intent = out
+[errflg]
+  standard_name = ccpp_error_flag
```

Comment:
Didn't this get changed to `ccpp_error_code`?

Reply:
As above.